scripts/development/docker-compose.yml

```diff
@@ -211,17 +211,17 @@ services:
       context: ../../frontend
       dockerfile: Dockerfile.dev
       args:
-        API_HOST: alpha-api
+        API_HOST: 172.20.0.4
```

Comment:
For consistency in the project do not change the API_HOST in the dev set-up. Pls reset to `alpha-api`.